### DISCUSSION TITLE
[Request] 如果用api调用lobechat dalle3插件

### DISCUSSION THREAD

**Discussion by yuanjie-ai** [lobehub/lobehub#1350] [Development Question] [answered]
### 🥰 需求描述

[Request] 如果用api调用lobechat dalle3插件

### 🧐 解决方案

[Request] 如果用api调用lobechat dalle3插件

### 📝 补充信息

[Request] 如果用api调用lobechat dalle3插件
![image](https://github.com/lobehub/lobe-chat/assets/20265321/fb00f292-1bf1-46ed-9b51-1cd39333b22a)

[a comment by an automated account is not shown]

**Comment by yuanjie-ai** [comment]
我想通过代码的方式，指定 function调用到lobechat相关插件

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
在这里： https://github.com/lobehub/lobe-chat/blob/main/src/app/api/openai/images/route.ts

但我们不保证稳定性，接口未来随时可能变动
